### PR TITLE
Feature/New vehicle data `WindowStatus`

### PR DESCRIPTION
**Pull request by Mariia-Novosiadla**
Guidelines updates according to issue [#3193](https://github.com/smartdevicelink/sdl_core/issues/3193).

 Proposal: [SDL-0261](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0261-New-vehicle-data-WindowStatus.md)

This PR is **ready** for review.

### Summary

1.  Updates for **Common**
* extending `VehicleDataType` with new element `VEHICLEDATA_WINDOWSTATUS`

2. Updates in **VehicleInfo**
* adding `windowStatus` to request/response params list of the following RPCs:
SubscribeVehicleData
UnsubscribeVehicleData
GetVehicleData
OnVehicleData

### CLA
* [x]  I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)s

### CODE REVIEW

**Comment by LitvinenkoIra** [comment]
@atiwari9 please review this PR

**Comment by atiwari9** [comment]
PR Approved.

**Comment by LitvinenkoIra** [comment]
@theresalech PR is ready for Livio review. Thank you!

File: docs/Common/Structs/index.md [hunk 1]
```diff
@@ -1130,3 +1130,17 @@ There are no defined parameters for this struct
...
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|approximatePosition|Integer|true|minvalue: 0<br>maxvalue: 100|The approximate percentage that the window is open -<br>0 being fully closed, 100 being fully open|
+|deviation|Integer|true|minvalue: 0<br>maxvalue: 100|The percentage deviation of the approximatePosition.<br>e.g. If the approximatePosition is 50 and the deviation is 10,<br>then the window's location is somewhere between 40 and 60|
+
+### WindowStatus
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|location|[Common.Grid](../sructs/#grid)|true|||
```
Comment (ShobhitAd) [inline docs/Common/Structs/index.md:1144]:
```suggestion
|location|[Common.Grid](../structs/#grid)|true|||
```

File: docs/Common/Structs/index.md [hunk 2]
```diff
@@ -1130,3 +1130,17 @@ There are no defined parameters for this struct
...
+|state|[Common.WindowState](../sructs/#windowstate)|true|||
```
Comment (ShobhitAd) [inline docs/Common/Structs/index.md:1145]:
```suggestion
|state|[Common.WindowState](../structs/#windowstate)|true|||
```

File: docs/VehicleInfo/GetVehicleData/index.md [hunk 3]
```diff
@@ -107,6 +108,7 @@ The HMI will have to update this field if the user chooses to reset this value (
 |engineOilLife|Float|false|minvalue: 0<br>maxvalue: 100|
 |electronicParkBrakeStatus|[Common.ElectronicParkBrakeStatus](../../common/enums/#electronicparkbrakestatus)|false||
 |cloudAppVehicleID|String|false||
+|windowStatus|[Common.WindowStatus](../../common/structs/#windowstatus)|false|array: true<br>minvalue: 0<br> maxvalue: 100|
```
Comment (ShobhitAd) [inline docs/VehicleInfo/GetVehicleData/index.md:111]:
```suggestion
|windowStatus|[Common.WindowStatus](../../common/structs/#windowstatus)|false|array: true<br>minsize: 0<br>maxsize: 100|
```

File: docs/VehicleInfo/OnVehicleData/index.md [hunk 4]
```diff
@@ -63,6 +63,7 @@ The HMI will have to update this field if the user chooses to reset this value (
 |engineOilLife|Float|false|minvalue: 0<br>maxvalue: 100|
 |electronicParkBrakeStatus|[Common.ElectronicParkBrakeStatus](../../common/enums/#electronicparkbrakestatus)|false||
 |cloudAppVehicleID|String|false||
+|windowStatus|[Common.WindowStatus](../../common/structs/#windowstatus)|false|array: true<br>minvalue: 0<br> maxvalue: 100|
```
Comment (ShobhitAd) [inline docs/VehicleInfo/OnVehicleData/index.md:66]:
```suggestion
|windowStatus|[Common.WindowStatus](../../common/structs/#windowstatus)|false|array: true<br>minsize: 0<br>maxsize: 100|
```

File: docs/Common/Structs/index.md (hunk 1 above)
Comment (Mariia-Novosiadla) [reply docs/Common/Structs/index.md]: fixed in a756cf9

File: docs/Common/Structs/index.md (hunk 2 above)
Comment (Mariia-Novosiadla) [reply docs/Common/Structs/index.md]: fixed in 9018e6b

File: docs/VehicleInfo/GetVehicleData/index.md (hunk 3 above)
Comment (Mariia-Novosiadla) [reply docs/VehicleInfo/GetVehicleData/index.md]: fixed in d54a22b

File: docs/VehicleInfo/OnVehicleData/index.md (hunk 4 above)
Comment (Mariia-Novosiadla) [reply docs/VehicleInfo/OnVehicleData/index.md]: fixed in 104ab54

File: docs/Common/Structs/index.md [hunk 5]
```diff
@@ -1130,3 +1130,16 @@ There are no defined parameters for this struct
...
+|:---|:---|:--------|:---------|:----------|
+|approximatePosition|Integer|true|minvalue: 0<br>maxvalue: 100|The approximate percentage that the window is open -<br>0 being fully closed, 100 being fully open|
+|deviation|Integer|true|minvalue: 0<br>maxvalue: 100|The percentage deviation of the approximatePosition.<br>e.g. If the approximatePosition is 50 and the deviation is 10,<br>then the window's location is somewhere between 40 and 60|
+
+### WindowStatus
+
+|Name|Type|Mandatory|Additional|Description|
+|:---|:---|:--------|:---------|:----------|
+|location|[Common.Grid](../structs/#grid)|true|||
+|state|[Common.WindowState](../structs/#windowstate)|true|||
```
Comment (ShobhitAd) [inline docs/Common/Structs/index.md:1145]: These links don't seem to work on the dev portal. Looking at the other structs, it seems we don't link to other structs within this page(See `grid` in `SeatLocation`, `seats` in `SeatLocationCapability`, etc.)

```suggestion
|location|Common.Grid|true|||
|state|Common.WindowState|true|||
```

Comment (Mariia-Novosiadla) [reply docs/Common/Structs/index.md]: fixed in 3cea6e5

**Comment by ShobhitAd** [comment]
All `WindowStatus` feature PRs have been approved. Please resolve the merge conflicts

**Pull request merged**